roles/pulp_api/README.md

```diff
@@ -10,7 +10,7 @@ Role Variables
   listen.](https://docs.gunicorn.org/en/stable/settings.html#bind)
   One can specify a unix socket path instead (recommended value is `'unix:/var/run/pulpcore-api/pulpcore-api.sock'`).
   Defaults to `'127.0.0.1:24817'`.
-* `pulp_api_workers`: Number of Pulp Content `gunicorn` processes for handling requests. Defaults to 1.
+* `pulp_api_workers`: Number of `gunicorn` processes for handling Pulp API requests. Defaults to 1.
```

Comment:
Doesn't this need a default entry and specification in the pulpcore-api service file?

Reply:
@bmbouter  It's already there, I didn't add it.  I'm just cleaning up the documentation.

Reply:
Oh that's right! Thanks. So then just the changelog and issue then.